docs/specs/examples/example-feature-x-spec.rst

```diff
@@ -0,0 +1,216 @@
+Example
+=======
```

Comment:
It seems people have been using Markdown files instead (but with the same structure as this doc); see https://github.com/bigchaindb/bigchaindb/tree/tendermint/proposals 

Maybe change to Markdown, just to reflect that apparent preference. Markdown is what GitHub issues and pull requests expect, so GitHub users generally know Markdown but not necessarily RST.